nss-cohort-37/DogWalkerMVC
# Mac nav - updated -

# Nav Bar
Built out the main nav bar

## How to Test 
on any page the nav bar should be displayed at the top. 
- hover over any nav item and they should change color to a light blue
- if you have any of your controllers built out, click the link and ensure that it takes you to the `index` of that controller

### Components Affected 
- site.css
- _Layout.cshtml
